src/java/main/org/apache/zookeeper/common/X509Util.java

```diff
@@ -199,4 +199,4 @@ public static X509TrustManager createTrustManager(String trustStoreLocation, Str
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Corrected the unnecessary change +}
Closing this pull req to raise new with corrected changes